src/KafkaFlow.LogHandler.Microsoft/MicrosoftLogHandler.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace KafkaFlow
+{
+    using System;
+    using System.Text.Json;
+    using Microsoft.Extensions.Logging;
+
+    internal class MicrosoftLogHandler : ILogHandler
+    {
+        private readonly ILogger logger;
+
+        public MicrosoftLogHandler(ILoggerFactory loggerFactory)
+        {
+            this.logger = loggerFactory.CreateLogger("KafkaFlow.LogHandler");
```

Comment:
> I would change the string from "KafkaFlow.LogHandler" to just "KafkaFlow". What do you think?
will do

>It would be nice to give the option to the client application pass this value too
can i address this with another PR later ?